### ISSUE TITLE
[Suggestion] Find the files by the owner

### ISSUE THREAD

**Issue by extend1994** [hackmdio/hackmd#653]
For example, when I need to some files owned by me. It sometimes takes me much time to find it.
Maybe it would be good to have a functionality of finding the files by the owner.
Or especially, owners can find the files they created. :)

**Comment by SISheogorath** [comment]
What kind of files are we talking about? Uploads?

If you search for pads, you should be able to find them on the history page.

**Comment by extend1994** [comment]
Ah, I should say "note", the markdown files on the history page. Sorry about confusing you :sweat: 
All notes including the ones created by others and oneself show on the history page.
It would not be easy if one only wants to see the notes created by oneself.
Hope I didn't miss something.

I just updated the comment and issue title.

**Comment by SISheogorath** [comment]
Oh yes, that's currently a missing feature but it'll maybe change with the upcoming release. We'll have folders to put notes into then, so I maybe this solves your problem. If you are fine with that, just wait for the next release which I try to get out to or after the next weekend

**Comment by extend1994** [comment]
Sure! @SISheogorath 
Just afraid of forgetting to propose this suggestion, no pressure :smiley:

**Comment by ccoenen** [comment]
This might also be related to https://github.com/hackmdio/hackmd/issues/314 - a profile page could easily list public pads of that user.

**Comment by SISheogorath** [comment]
Agreed. We should really put some effort into creating user management for the next major version

**Comment by dskrad** [comment]
I was under the impression that the "history" is just useful to see the last few pads created or accessed on your account. I thought you could remove items from history and still be able to search for them. The current paradigm is misleading. Removing from history does not remove the pad (still accessible if you have the url) but it effectively deletes it, because I have no way of searching by keyword or by tag. It is impossible to find it since the url is a random long alphanumeric string.

**Comment by ccoenen** [comment]
I would value your input in #952, a work-in-progress for a profile page. This would also list all your notes in my current proposal.
